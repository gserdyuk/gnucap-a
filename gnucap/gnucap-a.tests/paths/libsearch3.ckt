* test for library search 

v1 n1 0 dc 1
.include ./lib_file2.ckt
.include ./lib_file3.ckt

.list
.print op v(n*)
.op
.end
