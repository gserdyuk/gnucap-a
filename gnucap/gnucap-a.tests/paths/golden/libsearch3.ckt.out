Gnucap gnucap-arails  2009.12.07 RCS 26.136 experimental_1 2010-02-16
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
* test for library search 
v1 ( n1 0 )  DC  1.
v2 ( n2 0 )  DC  2.
v3 ( n3 0 )  DC  3.
#           v(n1)      v(n2)      v(n3)     
 27.        1.         2.         3.        
